src/traces/pointcloud/attributes.js

```diff
@@ -0,0 +1,124 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var scatterglAttrs = require('../scattergl/attributes');
+
+module.exports = {
+    x: scatterglAttrs.x,
+    y: scatterglAttrs.y,
+    xy: {
+        valType: 'data_array',
+        description: [
+            'Faster alternative to specifying `x` and `y` separately.',
+            'If supplied, it must be a typed `Float32Array` array that',
+            'represents points such that `xy[i * 2] = x[i]` and `xy[i * 2 + 1] = y[i]`'
+        ].join(' ')
+    },
+    indexid: {
+        valType: 'data_array',
+        description: [
+            'A sequential value, 0..n, supply it to avoid creating this array inside plotting.',
+            'Its length must be equal to or greater than the number of points.',
+            'For the best performance and memory use, create one large `indexid` typed array',
+            'that is guaranteed to be at least as long as the largest number of points during',
+            'use, and reuse it on each `Plotly.restyle()` call.'
+        ].join(' ')
+    },
+    bounds: {
```

Comment:
perhaps, splitting this into `xbounds` and `ybounds` would a little more plotly-esque.

cc @cldougl